### ISSUE TITLE
error time tracking when added to an issue

### ISSUE THREAD

**Issue by mckaygerhard** [go-gitea/gitea#5316]
- Gitea version (or commit ref): 1.5.3
- Git version: 2.11
- Operating system: Linux Debian
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [x] No
  - [ ] Not relevant
- Log gist:

```
2018/11/11 16:18:28 [I] [SQL] SELECT * FROM `user` INNER JOIN issue_assignees ON assignee_id = `user`.id WHERE (issue_assignees.issue_id = ?) []interface {}{1}
2018/11/11 16:18:28 [I] [SQL] SELECT COALESCE(sum(`time`),0) FROM `tracked_time` WHERE issue_id=? []interface {}{1}
2018/11/11 16:18:28 [I] [SQL] SELECT `id`, `type`, `issue_id`, `comment_id`, `user_id`, `created_unix` FROM `reaction` WHERE reaction.issue_id=? ORDER BY `reaction`.`issue_id` ASC, `reaction`.`comment_id` ASC, `reaction`.`created_unix` ASC, `reaction`.`id` ASC []interface {}{1}
2018/11/11 16:18:28 [I] [SQL] SELECT `id`, `assignee_id`, `issue_id` FROM `issue_assignees` WHERE `assignee_id`=? AND `issue_id`=? LIMIT 1 []interface {}{1, 1}
2018/11/11 16:18:28 [I] [SQL] SELECT `name` FROM `user` WHERE `id`=? LIMIT 1 []interface {}{2}
2018/11/11 16:18:28 [I] [SQL] INSERT INTO `tracked_time` (`issue_id`,`user_id`,`created_unix`,`time`) VALUES (?, ?, ?, ?) []interface {}{1, 1, 1541967508, 9180}
2018/11/11 16:18:28 [I] [SQL] BEGIN TRANSACTION
2018/11/11 16:18:28 [I] [SQL] INSERT INTO `comment` (`type`,`poster_id`,`issue_id`,`label_id`,`old_milestone_id`,`milestone_id`,`assignee_id`,`removed_assignee`,`old_title`,`new_title`,`commit_id`,`line`,`content`,`created_unix`,`updated_unix`,`commit_sha`) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) []interface {}{14, 1, 1, 0, 0, 0, 0, false, "", "", 0, 0, "2h 33min", 1541967508, 1541967508, ""}
2018/11/11 16:18:28 [I] [SQL] SELECT `id`, `lower_name`, `name`, `full_name`, `email`, `keep_email_private`, `passwd`, `login_type`, `login_source`, `login_name`, `type`, `location`, `website`, `rands`, `salt`, `language`, `created_unix`, `updated_unix`, `last_login_unix`, `last_repo_visibility`, `max_repo_creation`, `is_active`, `is_admin`, `allow_git_hook`, `allow_import_local`, `allow_create_organization`, `prohibit_login`, `avatar`, `avatar_email`, `use_custom_avatar`, `num_followers`, `num_following`, `num_stars`, `num_repos`, `description`, `num_teams`, `num_members`, `diff_view_style` FROM `user` WHERE `id`=? LIMIT 1 []interface {}{2}
2018/11/11 16:18:28 [I] [SQL] UPDATE `issue` SET `updated_unix` = ? WHERE `id`=? []interface {}{1541967508, 1}
2018/11/11 16:18:28 [I] [SQL] COMMIT
[Macaron] 2018-11-11 16:18:28: Completed POST /eeee/venenux-alone/issues/1/times/add 303 See Other in 642.204905ms
``` 

## Description

after created a issue with default tags and a milestrone, try to added time tracking as 2 hours, got a Error 324 (net::ERR_EMPTY_RESPONSE): in the browser, using chrome

## Screenshots

![errorgitea](https://user-images.githubusercontent.com/1571724/48317891-71441880-e5f0-11e8-91e7-8f23a6e81f0e.gif)

**Comment by mckaygerhard** [comment]
now after happened the bug the instalation cannot show any more at the browsers

**Comment by mckaygerhard** [comment]
i got it, after added the timetracking, the gitea tries to redirect to the configured domain url event the agnostc path.. so if we are outside with NAT will show localhost or whatever configured!

**Comment by lunny** [comment]
So you have to config your app.ini to set that with a outside NAT domain or IP.

**Comment by mckaygerhard** [comment]
noT, the app must make same as for rest of thigs, not use hard paths, due it's stupid if i inside the network (that are 100/1000 ) try to do with the external domain that are more slow!

when i touch the liabels, milestones etc, the gitea uses the current path url, not the hardcoded path.. specially if i foun it behing a reverse proxy

**Comment by mckaygerhard** [comment]
@lunny this are not a question, its a bug as same bug as in #5425 due many hardcoded path to a hijacked web root of the assumed web werver of the installations

**Comment by mckaygerhard** [comment]
this bug are causing issues in others due root url are kind of hardcoded from ROOT_URL as i described and today it 1.6.2 release and the bug is still happened!

**Comment by lunny** [comment]
@mckaygerhard are you using a sub url configuration?

**Comment by mckaygerhard** [comment]
yeah @lunny a subpath. not subdomian neither a complete hijacket webroot

**Comment by zeripath** [comment]
Please could you explain very carefully what the problem is, your configuration - including if you are proxying and how you are doing it. Please give us some example URLs, and if it works for some URLs and not others please show us those.

**Comment by mckaygerhard** [comment]
stale bot marked in two weeks as closed! today 1.7.4 and no solution?
